skt/reporter.py

```diff
@@ -350,11 +350,33 @@ def getmergeinfo(self):
         return result
 
     def get_kernel_config(self, suffix=None):
-        cfgname = "config.gz" if not suffix else "config_{}.gz".format(suffix)
+        """Add the kernel config to the list of attachments.
 
-        self.attach.append((cfgname, gzipdata(self.mergedata["config"])))
-        return ['\nThe kernel was built with the attached configuration '
-                '(%s).' % cfgname]
+        Args:
+            suffix: An optional filename suffix to add to the config file
+                    name. This is helpful for reporting config files for
+                    different kernel architectures.
+
+        Returns:
+            If the 'config' item exists in self.mergedata, the method returns
+            a new line for the report as a single item within a list. If the
+            'config' item does not exist, the method returns an empty list
+            and no extra lines are added to the report.
```

Comment:
This is a good explanation, thank you. Although I would drop the last part - the "and no extra lines are added to the report" phrase. Because it is not for this function to decide what gets added to the report and what doesn't. It certainly makes no reports itself.